EditorExtensions/Settings/WESettings.cs

```diff
@@ -369,6 +369,12 @@ public sealed class JavaScriptSettings : LinterSettings<JavaScriptSettings>, IMi
         [Description("Forces all rules to be terminated with semicolons when true.")]
         [DefaultValue(true)]
         public bool TermSemicolons { get; set; }
+
+		[Category("Minification")]
+		[DisplayName("Custom base url for resources")]
+		[Description("Specifies a custom base url in case your application is not running directly on the root directory.")]
```

Comment:
IMO, first it should be categorized as bundle:

```c#
[Category("Bundle")]
```

(and would be nice to investigate whether it is applicable to .ts, .html and .css bundles as well, if yes then it should be added to `interface IBundleSettings` as well)

Secondly, would something like this make sense:

> Specifies a base path to be prepended in bundled .js, in case the assets are hosted from a virtual directory.